File: FastMines_android/src/main/java/fmg/android/app/model/items/LogoDataItem.java
package fmg.android.app.model.items;

import fmg.android.img.AndroidBitmapView;
import fmg.android.img.Logo;
import fmg.core.img.LogoModel;

/** Logo as data model */
public class LogoDataItem extends BaseDataItem<Void, LogoModel, AndroidBitmapView<LogoModel>, Logo.LogoAndroidBitmapController> {

    public LogoDataItem() {
        super(null);
        setTitle("Mosaics");
    }

    @Override
    public Logo.LogoAndroidBitmapController getController() {
        if (this.controller == null) {
            var ctrl = new Logo.LogoAndroidBitmapController();
            var m = ctrl.getModel();
            m.setBorderWidth(3);

            // TODO m.setRotateMode(LogoModel.ERotateMode.color);
            m.setRotateAngle(10);
            m.setUseGradient(true);

            setController(ctrl);
        }
        return this.controller;
    }

}

File: FastMines_uwp/FastMines.Uwp.Main/DataModel/Items/LogoDataItem.cs
using Microsoft.Graphics.Canvas;
using Fmg.Common;
using Fmg.Core.Img;
using LogoView       = Fmg.Uwp.Img.Win2d.Logo.CanvasBmpView;
using LogoController = Fmg.Uwp.Img.Win2d.Logo.CanvasBmpController;

namespace Fmg.DataModel.Items {

    /// <summary> Logo as data model </summary>
    public class LogoDataItem : BaseDataItem<Nothing, LogoModel, LogoView, LogoController> {

        public LogoDataItem()
            : base(null)
        {
            Title = "Mosaics";
        }

        public override LogoController Entity {
            get {
                if (entity == null) {
                    var tmp = new LogoController(CanvasDevice.GetSharedDevice());
                    var m = tmp.Model;
                    m.BorderWidth = 3;
                    m.RotateMode = LogoModel.ERotateMode.Color;
                    Entity = tmp; // call this setter
                }
                return entity;
            }
        }

    }

}
